src/Aspire.Dashboard/wwwroot/js/app.js

```diff
@@ -53,15 +53,33 @@ function isScrolledToBottom(container) {
 }
 
 window.copyTextToClipboard = function (id, text, precopy, postcopy) {
-    let tooltipDiv = document.querySelector('fluent-tooltip[anchor="' + id + '"]').children[0];
+    const button = document.querySelector(`#${id}`);
```

Comment:
Is this equivalent to the slightly simpler:

```suggestion
    const button = document.getElementById(id);
```